### PR TITLE
Recipe coverts + fixes + makeover of food and drink

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Goes through every food item, every drink bottle and fixes the desc to not be bay/tg/agressive or tell you what you think/feel/how to act
Fixes beet soup CWJ code error
Fixes mushrooms for CWJ 
Converts a few more dishes to CWJ

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks/burger.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 /obj/item/reagent_containers/food/snacks/cheeseburger
 	name = "cheeseburger"
-	desc = "The cheese adds a good flavor."
+	desc = "A normal burger apart form the addition of a yellow square of cheese."
```
Comment (DRD21) [inline code/modules/reagents/reagent_containers/food/snacks/burger.dm:30]: from, not form

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 2]
```diff
@@ -345,14 +345,14 @@
...
-	desc = "A bottle of pure Fernet Bronca, imported from Cordoba Space Station."
+	desc = "A bottle of pure Fernet Bronca."
 	icon_state = "fernetbottle"
 	center_of_mass = list("x"=16, "y"=4)
 	preloaded_reagents = list("fernet" = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/neulandschnapps
 	name = "Neuland Himbeergeist"
-	desc = "A kriosan-approved spirits covered in german text, wax stamp on the bottle with the crest of a obscure and minor Castellan Lord. Time to shout 'Prost!' Ja?"
+	desc = "A kriosan-approved spirits covered in kriosan text, wax stamp on the bottle with the crest of a obscure and minor Castellan Lord. Time to shout 'Prost!' Ja?"
```
Comment (DRD21) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:355]: Kriosan, not kriosan

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 3]
```diff
@@ -214,50 +214,50 @@
...
+	desc = "A brand of coffee-flavoured liqueur."
 	icon_state = "kahluabottle"
 	center_of_mass = list("x"=17, "y"=3)
 	preloaded_reagents = list("kahlua" = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/goldschlager
-	name = "College Girl Goldschlager"
-	desc = "Because they are the only ones who will drink 100 proof cinnamon schnapps."
+	name = "John Prince's Goldschlager"
+	desc = "A brand of goldschlager branded."
```
Comment (DRD21) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:246]: branded.... what?

File: code/modules/reagents/reagent_containers/food/snacks/burger.dm [hunk 4]
```diff
@@ -65,7 +65,7 @@
 
 /obj/item/reagent_containers/food/snacks/baconburger
 	name = "bacon burger"
-	desc = "What makes a burger even more perfect? The addition of bacon, of course."
+	desc = "A normal burger apart form some strips of bacon added."
```
Comment (DRD21) [inline code/modules/reagents/reagent_containers/food/snacks/burger.dm:68]: from, not form

**Comment by cdb-is-not-good** [comment]
was gonna do a big long code review but don't really got the energy for that. So instead; 
Changing perspective from first person to be more vague = good
Taking flavor out of stuff unnecessarily in favor of sanitizing it = bad

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 5]
```diff
@@ -379,7 +379,7 @@
 
 /obj/item/reagent_containers/food/drinks/flask/vacuumflask
 	name = "vacuum flask"
-	desc = "Keeping your drinks at the perfect temperature since 1892."
+	desc = "A flask for keeping drinks eather colder or hotter for longer."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/drinks.dm:382]: "Either" not "eather"

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 6]
```diff
@@ -436,7 +436,7 @@
 
 /obj/item/reagent_containers/food/snacks/cookie
 	name = "cookie"
-	desc = "COOKIE!!!"
+	desc = "cookie!"
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:439]: Missing capitalization on the "C"

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 7]
```diff
@@ -846,7 +845,7 @@
 
 /obj/item/reagent_containers/food/snacks/popcorn
 	name = "popcorn"
-	desc = "Now let's find some cinema."
+	desc = "A light snack of corn seeds that have been heated in oil untill exploided."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:848]: I suggest changing this to "A light snack of corn seed heated in oil until puffed."

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 8]
```diff
@@ -949,7 +948,7 @@
 
 /obj/item/reagent_containers/food/snacks/badrecipe
 	name = "burned mess"
-	desc = "Someone should be demoted from chef for this."
+	desc = "A misstake of cooking."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:951]: "A cooking mistake"

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 9]
```diff
@@ -1109,7 +1108,7 @@
 
 /obj/item/reagent_containers/food/snacks/enchiladas
 	name = "enchiladas"
-	desc = "Viva La Mexico!"
+	desc = "Rolled meat chili and corn into one."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1111]: "Meat, chili and corn rolled into a tortilla"

or "Meat, chili and corn rolled in one tortilla."

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 10]
```diff
@@ -1237,7 +1236,7 @@
 
 /obj/item/reagent_containers/food/snacks/ricepudding
 	name = "rice pudding"
-	desc = "Where's the jam?"
+	desc = "A rice base pudding."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1239]: "A rice based pudding."

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 11]
```diff
@@ -1312,7 +1311,7 @@
 
 /obj/item/reagent_containers/food/snacks/meatballspagetti
 	name = "spaghetti & meatballs"
-	desc = "Now thats a nic'e meatball!"
+	desc = "Speghetti served with meatballs."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1314]: "Spaghetti"

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 12]
```diff
@@ -1327,7 +1326,7 @@
 
 /obj/item/reagent_containers/food/snacks/spesslaw
 	name = "spesslaw"
-	desc = "A lawyers favourite."
+	desc = "Meatballs served with speghetti."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1329]: "Spaghetti"

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 13]
```diff
@@ -1355,7 +1354,7 @@
 
 /obj/item/reagent_containers/food/snacks/candiedapple
 	name = "candied apple"
-	desc = "An apple coated in sugary sweetness."
+	desc = "An apple coated in liquidifed sugar before being cooled."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1357]: "Liquidified"

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 14]
```diff
@@ -1449,7 +1448,7 @@
 
 /obj/item/reagent_containers/food/snacks/appletart
 	name = "golden apple streusel tart"
-	desc = "A tasty dessert that won't make it through a metal detector."
+	desc = "A dessert made from golden apples rather then normal apples."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1451]: "than"

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 15]
```diff
@@ -1515,7 +1514,7 @@
 
 /obj/item/reagent_containers/food/snacks/sliceable/tofubread
 	name = "tofubread"
-	icon_state = "Like meatbread but for vegetarians. Not guaranteed to give superpowers."
+	icon_state = "A bread meatbread made from tofu rather then meat."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1517]: "than"

File: code/modules/reagents/reagent_containers/food/snacks/cakes_pies.dm [hunk 16]
```diff
@@ -389,7 +389,7 @@
 	name = "meat pie"
 	icon_state = "meatpie"
 	taste_tag = list(MEAT_FOOD, FLOURY_FOOD)
-	desc = "An old barber recipe, very delicious!"
+	desc = "A pie stuffed with meats rather then fruits or vegetables."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks/cakes_pies.dm:392]: "than"

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 17]
```diff
@@ -150,40 +125,20 @@
...
-	icon_state = "xenomeat"
-	filling_color = "#E2FFDE"
-	//Queens aren't hard to kill and aren't too rare. 6 units of menace should be enough per slab.
-	bitesize = 6
 	preloaded_reagents = list("protein" = 8, "pararein" = 8, "menace" = 6, "ammonia" = 4)
 
 /obj/item/reagent_containers/food/snacks/meat/carp
 	name = "carp fillet"
-	desc = "A juicy fillet cut from a carp. The potent and powerful venom they produce just gives it a unique tang. Delicious!"
+	desc = "A juicy fillet cut from a carp. The known for its potent and powerful venom they produce."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm:141]: "They are known for the potent and powerful venom they produce."

File: code/modules/reagents/reagent_containers/food/snacks/pizza.dm [hunk 18]
```diff
@@ -82,7 +82,7 @@
 
 /obj/item/reagent_containers/food/snacks/sliceable/pizza/vegetablepizza
 	name = "vegetable pizza"
-	desc = "No Tomato Sapiens were harmed during the baking of this pizza."
+	desc = "A pizza made with vegatables to replace cheese."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks/pizza.dm:85]: A vegetable pizza does not replace the cheese, it replaces the meat.

"A pizza made with vegetables to replace meat."

File: code/modules/reagents/reagent_containers/food/snacks/sandwich.dm [hunk 19]
```diff
@@ -117,7 +117,7 @@
 
 /obj/item/reagent_containers/food/snacks/sandwich
 	name = "sandwich"
-	desc = "A grand creation of meat, cheese, bread, and several leaves of lettuce! Arthur Dent would be proud."
+	desc = "A simple snadwich of meat, cheese, bread, and several leaves of lettuce."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks/sandwich.dm:120]: "Sandwich"

File: code/modules/reagents/reagent_containers/food/snacks/sandwich.dm [hunk 20]
```diff
@@ -218,7 +218,7 @@
 
 /obj/item/reagent_containers/food/snacks/cubancarp
 	name = "cuban carp"
-	desc = "A sandwich that burns your tongue and then leaves it numb!"
+	desc = "A spicy sandwich that has been tasted."
```
Comment (Smokingsquirrel) [inline code/modules/reagents/reagent_containers/food/snacks/sandwich.dm:221]: I think you meant "toasted" here.

File: code/modules/cooking_with_jane/recipes/recipe.dm [hunk 21]
```diff
@@ -1438,11 +1533,12 @@
...
-		list(CWJ_ADD_PRODUCE, "mushrooms", qmod=0.2, reagent_skip=TRUE),
-		list(CWJ_ADD_PRODUCE, "mushrooms", qmod=0.2, reagent_skip=TRUE),
-		list(CWJ_ADD_PRODUCE, "mushrooms", qmod=0.2, reagent_skip=TRUE),
-		list(CWJ_ADD_PRODUCE, "mushrooms", qmod=0.2, reagent_skip=TRUE),
+		//Mushroom pizza is uniquic in that it takes any mushroom rather then a required type
+		list(CWJ_ADD_PRODUCE, "mushroom", qmod=0.2, reagent_skip=TRUE),
+		list(CWJ_ADD_PRODUCE, "mushroom", qmod=0.2, reagent_skip=TRUE),
+		list(CWJ_ADD_PRODUCE, "mushroom", qmod=0.2, reagent_skip=TRUE),
+		list(CWJ_ADD_PRODUCE, "mushroom", qmod=0.2, reagent_skip=TRUE),
+		list(CWJ_ADD_PRODUCE, "mushroom", qmod=0.2, reagent_skip=TRUE),
```
Comment (Trilbyspaceclone) [inline code/modules/cooking_with_jane/recipes/recipe.dm:1541]:
```suggestion
		list(CWJ_ADD_PRODUCE, "plump helmet", qmod=0.2, reagent_skip=TRUE),
		list(CWJ_ADD_PRODUCE, "plump helmet", qmod=0.2, reagent_skip=TRUE),
		list(CWJ_ADD_PRODUCE, "plump helmet", qmod=0.2, reagent_skip=TRUE),
		list(CWJ_ADD_PRODUCE, "plump helmet", qmod=0.2, reagent_skip=TRUE),
		list(CWJ_ADD_PRODUCE, "plump helmet", qmod=0.2, reagent_skip=TRUE),
```

**Pull request merged**